src/Dapr.Actors/DaprHttpInteractor.cs

```diff
@@ -133,14 +136,13 @@ HttpRequestMessage RequestFunc()
             }
 
             var retval = await this.SendAsync(RequestFunc, relativeUrl, cancellationToken);
-
+            var header = "";
             IActorResponseMessageHeader actorResponseMessageHeader = null;
             if (retval != null && retval.Headers != null)
             {
                 if (retval.Headers.TryGetValues(Constants.ErrorResponseHeaderName, out IEnumerable<string> headerValues))
                 {
-                    var header = headerValues.First();
-
+                    header = headerValues.First();
```

Comment:
We should be getting the header info by it's name since we know there's more than one now.